haskell/test/Lorentz/Contracts/Test/Management.hs

```diff
@@ -301,7 +307,35 @@ managementSpec originate = do
 
         mgmContractPaused err
 
+  describe "Minter limit check" $ do
+    -- Currently the contract allows to add upto 12 minters.
+    let
+      configureMinterParam :: Int -> ConfigureMinterParam
+      configureMinterParam i =
+        ( #minter .! (KeyAddress $ hashKey  $ toPublic $ detSecretKey $ encodeUtf8 @Text ("a" <> show i))
+        , ( #current_minting_allowance .! Nothing
+          , #new_minting_allowance .! 20
+          ))
+
+    it "Can add minter until minter limit" $ integrationalTestExpectation $ do
+
+      withOriginated originate defaultOriginationParams $ \stablecoinContract -> do
+        withSender testMasterMinter $ do
+          mapM_ (lCallEP stablecoinContract (Call @"Configure_minter")) (configureMinterParam <$> [1..12])
+
+          lExpectStorage stablecoinContract $ \case
+            (StorageMinters minters)
+              | M.size(minters) /= 12  ->
```

Comment:
`M.size(minters)` looks like C++ :smiley: 

Why not `M.size minters`?

Also I think it can be made a bit shorter using `when` or `unless`, but it's a minor thing.

Reply:
>M.size(minters) looks like C++ smiley
>Why not M.size minters?

Probably hangover from LIGO.

>Also I think it can be made a bit shorter using when or unless

Will do..